nautobot_floor_plan/forms.py

```diff
@@ -71,6 +85,8 @@ class FloorPlanTileForm(NautobotModelForm):
     rack = DynamicModelChoiceField(
         queryset=Rack.objects.all(), required=False, query_params={"nautobot_floor_plan_floor_plan": "$floor_plan"}
     )
+    x_origin = forms.CharField(validators=[])
+    y_origin = forms.CharField()
```

Comment:
Is there a reason why `x_origin` has `validators=[]` and `y_origin` doesn't?

Reply:
leftover ... removed